.stylelintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "processors": ["stylelint-processor-styled-components"],
+  "extends": [
+    "stylelint-config-recommended",
+    "stylelint-config-styled-components"
+  ]
+}
```

Comment:
Just a note: we typically have adopted the practice of having our editors automatically inserting a newline at the end of each file (in viz, we actually enforce the `eol-last` rule, and I assume we will be pulling that over to this repo as well)

Reply:
I was looking for this in the viz .eslintrc for this, but found it was being enforced in an .editorconfig file, so I'm adding that file to blip -- there's nothing in there that I think would negatively affect the rest of blip, but let me know if I'm wrong there

Reply:
That should be fine 👍